packages/next/src/client/router.ts

```diff
@@ -140,10 +146,6 @@ export function useRouter(): NextRouter {
   return router
 }
 
-// INTERNAL APIS
-// -------------
-// (do not use following exports inside the app)
-
```

Comment:
Not sure if this should be removed - seems like someone wanted to make sure these don't accidentally get exported?

Reply:
This comment is used pretty inconsistently across our codebase. The solution is not to push internal APIs at the end of the file and _maybe_ remember it, but mark any internally exported functions with `@internal` which will strip the types during build.